### DISCUSSION TITLE
New RAP API from NRCS & WLFW

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-tools#259] [VBET]
## Backdrop
One of the features that we have lacked is the ability to look at NDVI changes and derivatives through time (e.g. BRAT and RCAT just use LANDFIRE slices to do what they do). One of the things that VBET gives us is a solid map of the part of the landscape we want to track changes in (i.e. riverscapes). One of the things that [Low-Tech Explorer 2.0](https://bda-explorer.herokuapp.com/) (being scoped now) will give us is where people are doing what in these riverscapes. 
![image](https://user-images.githubusercontent.com/9723905/110716289-158bf900-81c4-11eb-8a6c-268780b5f8f9.png)

A missing piece for effectiveness monitoring is to see what changes to these riverscapes are attributable to our changes. Riparian vegetation are big parts of that.

## Rangeland Analysis Platform
The [RAP](https://rangelands.app/) allows you for anywhere west of the Mississippi to view and summarize LANDSAT derivatives annually from 1984 to 2019. These include:
- Vegetation
  - [Cover](https://rangelands.app/products/#cover) (e.g. Perennial Forb & Grass Cover; Annual Forb & Grass Cover; Shrubs, Trees, Bare Ground) 
  - [Biomass](https://rangelands.app/products/#biomass) (e.g. Herbaceous, Annual Forb & Grass, Perennial Forb & Grass)
  - Fire Boundaries
- Sagebrush Conservation
  - Annual herbaceous cover
  - [Ecosystem Resilience](https://rangelands.app/products/#ecosystem-resilience-and-resistance)
  - [Categorical Tree Cover](https://rangelands.app/products/#categorical-tree-cover)
- Great Plains Conservation
  - [Woody Transitions](https://rangelands.app/products/#woody-transitions)
  - [Cultivation Risk](https://rangelands.app/products/#cultivation-risk)
![image](https://user-images.githubusercontent.com/9723905/110716827-26893a00-81c5-11eb-92c6-868385ea6c0c.png)

The interactivity is useful for us to think about as we scope the VBET Commercial Grade experience (@philipbaileynar and @shelbysawyer ), but also useful for us to think about how useful it would be for a big part of our audience to be able to pass their VBET polygons to this, or us provide the same functionality within our web portal. 

They also have layers like [Carbon estimates](https://rangelands.app/products/#rangeland-carbon), which might become helpful for us to think about using for carbon sequestration valuation.

## New RAP API

Brady just put out a [RAP API](https://rangelands.app/api/v1/) for the cover and production. These return GeoJSON for those areas with summaries of the values.

You provide a geoJSON as input representing the input (e.g. point, polyline or polygon). I'm thinking we do it for our valley bottom segment (of interest). 

----------------

# Questions
1. @MattReimer and @philipbaileynar, can you get an API key from Brady and see if we can get back this and suggest how and where we would store it? Would we put it in its own riverscapes project, in a VBET. Would we leave it as its geoJSON or suck it into a 
2. @shelbysawyer we should think about how we might display and summarize this information for a user for their riverscape of interest. What would the map layer(s) and views be? What would we graph or plot (e.g. a time series)? How would we interpret that?
3. @philipbaileynar I'm thinking this is a helpful thought exercise for how to connect to some "added value" for my little riverscape. In other words, give me something for the riverscape I care about (< 5 mi) and make my life easier so I don't need to go to that other place to get it.  I am not thinking about this as a new "network extent" output in Riverscapes Context or VBET or its own. Is that right? 
4. @philipbaileynar  they do have WMTS for their layers. Should/could we add these as additional "Basemaps" in RAVE?
5. @MattReimer, if they have WMTS for their layers, can we use that to display in our web maps (I'm guessing hard no)?

**Comment by joewheaton** [comment]
On question 4 that no-one answered, can @wally-mac and @tyler1218hatch please consider this one?

**Comment by wally-mac** [comment]
Tyler can you please take this on? Thanks

On Wed, Apr 7, 2021 at 10:17 AM Joe Wheaton ***@***.***>
wrote:

> On question 4 that no-one answered, can @wally-mac
> <https://github.com/wally-mac> and @tyler1218hatch
> <https://github.com/tyler1218hatch> please consider this one?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Riverscapes/riverscapes-tools/discussions/259#discussioncomment-580732>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFHZAUXZ33GGTYIBGQA4MS3THSAQ5ANCNFSM4Y7GJ3FQ>
> .
>
-- 
Sent from Gmail Mobile
Sorry for typos and brevity courtesy of my phone.

**Reply by tyler1218hatch** [reply]
@wally-mac  Yes, I'll look into this.
